client/src/components/users/types.ts

```diff
@@ -140,8 +147,16 @@ export class UserCollection {
     return this.users.filter((user) => user.isSelected && this.userIsVisible(user));
   }
 
+  isUserInfoSelectable(user: UserInfo): boolean {
```

Comment:
It's a bit confusing to have two function depending on the type, especially when those types are basically the same and the only thing preventing you from using the same function for both is the `isCurrent` attribute.
From what I'm seeing, the function is only used in `openUserContextMenu` which takes a `UserInfo` as a parameter, but the users display (grid/list) manipulate UserModel and emit signals with UserModel.
Which means that if you switch the type in `openUserContextMenu` to `UserModel` you can remove this function.